Makefile

```diff
@@ -106,6 +114,10 @@ unit-tests:
 	@echo $(CYAN)"# Testing the HaCl* shared library"$(NORMAL)
 	$(MAKE) -C snapshots/hacl-c unit-tests
 
+unit-tests-unrolled:
```

Comment:
This should definitely not be at top-level but handled in `test/Makefile`

Reply:
Well, if we want to move NSS to master these targets have to go somewhere. What's your suggestion where to put them?

Reply:
As mentioned before: `test/Makefile`

Reply:
obviously... 😬